<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1111</width>
      <height>781</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(189, 226, 176)</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="welcome">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(189, 226, 176)</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>1101</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 86pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)</string>
      </property>
      <property name="text">
       <string>Welcome To </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>1111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 73pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)
</string>
      </property>
      <property name="text">
       <string>GeoDoodle</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>510</y>
        <width>1111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 33pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}







</string>
      </property>
      <property name="text">
       <string>Click to Continue...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="location">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(176, 199, 234)</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1111</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 50pt &quot;Copperplate&quot;;
color:rgb(81, 141, 72)</string>
      </property>
      <property name="text">
       <string>Please select your location </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>1111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 25pt &quot;Copperplate&quot;;
color: black</string>
      </property>
      <property name="text">
       <string>Note that for the MVP, </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>1111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 25pt &quot;Copperplate&quot;;
color: black</string>
      </property>
      <property name="text">
       <string>This application will only work for Downtown Toronto </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>470</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>420</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 15pt &quot;Copperplate&quot;;
color: black</string>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>590</y>
        <width>1111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(95, 152, 80);
     border-radius: 10px;
     font: 40pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(185, 205, 237)
}


</string>
      </property>
      <property name="text">
       <string>NEXT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Radius">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>-150</x>
        <y>90</y>
        <width>1261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 30pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)</string>
      </property>
      <property name="text">
       <string>Please select your desired radius</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>320</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 18pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)</string>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="suffix">
       <string/>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>1111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 33pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}







</string>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="picture">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(189, 226, 176)</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1091</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 35pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)
</string>
      </property>
      <property name="text">
       <string>Please choose from the </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>1121</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 35pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)
</string>
      </property>
      <property name="text">
       <string> given images below!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="cartoon1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 18pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}

</string>
      </property>
      <property name="text">
       <string>Irregular Pentagon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cartoon5">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>440</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 18pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}

</string>
      </property>
      <property name="text">
       <string>Regular Rhombus</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cartoon4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>440</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 18pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}

</string>
      </property>
      <property name="text">
       <string>Irregular Hexagon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cartoon3">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>270</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 18pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}

</string>
      </property>
      <property name="text">
       <string>Square</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cartoon2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>251</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 18pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}

</string>
      </property>
      <property name="text">
       <string>Triangle</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="load">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(176, 199, 234)</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>1101</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 80pt &quot;Copperplate&quot;;
color:rgb(81, 141, 72)</string>
      </property>
      <property name="text">
       <string>Loading...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>520</y>
        <width>1111</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 30 25pt &quot;Copperplate&quot;;
color:rgb(185, 205, 237);
background-color: rgb(95, 152, 80);</string>
      </property>
      <property name="text">
       <string>Give us a second while we generate your route!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="output">
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>1111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 35pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)
</string>
      </property>
      <property name="text">
       <string>Here is the route based </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>1101</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 35pt &quot;Copperplate&quot;;
color: rgb(5, 116, 186)
</string>
      </property>
      <property name="text">
       <string>on the route that you chose!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="output_url">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>306</y>
        <width>1111</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 75 18pt &quot;Copperplate&quot;;
color: rgb(0, 0, 0)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="restartButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>540</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
     background-color: rgb(5, 116, 186);
     border-radius: 10px;
     font: 18pt &quot;.AppleSystemUIFont&quot;;
     color: rgb(198, 230, 184);
}


</string>
      </property>
      <property name="text">
       <string>Restart</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1152</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
